Source/santad/EventProviders/SNTEndpointSecurityManager.mm

```diff
@@ -84,9 +98,7 @@ - (void)establishClient API_AVAILABLE(macos(10.15)) {
         if (m->action_type == ES_ACTION_TYPE_AUTH) {
           es_respond_auth_result(self.client, m, ES_AUTH_RESULT_ALLOW, false);
         }
-        if (self.selfPID != pid) {
```

Comment:
Can we remove selfPID now? It looks like it's still referred to but those cases might also be removable?

Reply:
+1 for removing these. However, we would need to bail if `-[muteSelf:]` fails.

Reply:
I'm all for bailing in cases like these instead of trying to carry on. Simplifies a bunch of logic.